lua/autorun/lfs_basescript.lua

```diff
@@ -935,25 +935,17 @@ if CLIENT then
 		outline = false,
 	} )
 
-	function simfphys.LFS.HudPaintPlaneIdentifier( X, Y, In_Col, target_ent )
-		if not IsValid( target_ent ) then return end
+	function simfphys.LFS.HudPaintPlaneIdentifier( targetEnt, x, y, r, g, b, a )

```

Comment:
**Bigger change here**

The actual method itself only needs the raw rgb values, so ideally we wouldn't pass color objects into it.

I think to maintain backwards compatibility I'll make either a wrapper, or a new function. TODO on that.